### PR TITLE
sigproc.ewm - Fixes #47 by expanding dims of scale_arr

### LINKED ISSUE #47: Enhancement: EWMA should work with arrays with ndim > 2
The following snippet right above the `def ewma` can make it work, but I had putting a while loop inside a function that gets called frequently. I'm sure there's a better way.

```python
            while scale_arr.ndim < buffer_data.ndim:
                scale_arr = scale_arr[..., None]
```

And this would have to be accompanied by removing the `, np.newaxis]` slicing on scale_arr within the function.

### PR DESCRIPTION
**Pull request by cboulay**
fixes #47 - scale_arr now has same ndim as buffered data, which allows for > 2-dim input.

### CODE REVIEW

**Comment by griffinmilsap** [comment]
I'd like to have a test written before merging this one; I'll take a crack at that in the coming days.  I suspect this will function just fine, but this unit is quite complicated and used in some important places for us at the moment.

File: extensions/ezmsg-sigproc/ezmsg/sigproc/ewmfilter.py [hunk 1]
```diff
@@ -73,9 +73,12 @@ async def sync_output(self) -> AsyncGenerator:
             buffer_data = buffer.data
             buffer_data = np.moveaxis(buffer_data, axis_idx, 0)
 
+            while scale_arr.ndim < buffer_data.ndim:
```
Comment (pperanich) [inline extensions/ezmsg-sigproc/ezmsg/sigproc/ewmfilter.py:76]: Could also change this to
```python
if scale_arr.ndim < buffer_data.ndim:
    scale_arr = np.expand_dims(scale_arr, list(range(scale_arr.ndim, buffer_data.ndim)))
```

Comment (cboulay) [reply extensions/ezmsg-sigproc/ezmsg/sigproc/ewmfilter.py:76]: Definitely happy to get rid of a while loop!
Let me know if you're interested in merging this PR in theory then I'll make the requested edits.

Comment (pperanich) [reply extensions/ezmsg-sigproc/ezmsg/sigproc/ewmfilter.py:76]: It all looks good to me. I'll defer to @griffinmilsap if he'd really like to get some tests in place prior to making changes.

**Pull request merged**